src/schemas/index.js

```diff
@@ -32,11 +32,15 @@ export const schema = {
     },
     'host': {
       'type': 'link',
-      'target': '/header/info/host'
+      'target': '/header/host/host'
     },
     'basePath': {
       'type': 'link',
-      'target': '/header/info/basePath'
+      'target': '/header/host/basePath'
```

Comment:
Link fields transfer the value from another part of the form. They're useful so that we can deviate from the Swagger structure without having a separate post-processor.
I've found that separate post-processors can easily become cluttered and difficult to read, so I decided to integrate it into the form system.
The `target` field in link-type form fields tells what form field the value should be taken from.
The Swagger spec uses mixed camel case, so I don't think hyphens are a good idea for form field names/references.